doc/user_guide/installation.rst

```diff
@@ -1,27 +1,75 @@
 .. _installation:
 
 Installation
+============
+
+Command line
 ------------
 
+Pylint is installable using a package manager. Your package manager will find a version that
+works with your interpreter. We recommend ``pip``:
+
+.. code-block:: sh
+
+   pip install pylint
+
+
+Use the newer python interpreters if you can. It's possible to analyse code written
+for older interpreters by using ``py-version`` and setting it to the old interpreter. For
+example you can check that there are no ``f-string`` in python 3.5 code using python 3.8
+with an up-to-date pylint even if python 3.5 is past enf of life (EOL).
```

Comment:
```suggestion
It is recommend to use the newest Python interpreter you can use. However, it is also possible to analyse code written
for older interpreters by using the ``py-version`` option and setting it to the old interpreter. For
example you can check that there are no ``f-strings`` in Python 3.5 code using Python 3.8
with an up-to-date pylint even if Python 3.5 is past end of life (EOL).
```

Reply:
I think we should not try to advise users on the interpreter they should use for **their code** but stay in our lane an tell them what interpreter to use for pylint. (It's probably going to be the same than their code though let's be realist.)